blueprints/test-framework-detector.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(blueprint) {
     if ('ember-cli-qunit' in dependencies) {
       type = 'qunit';
     } else if ('ember-cli-mocha' in dependencies) {
-      var checker = new VersionChecker({ project: this.project });
+      var checker = new VersionChecker({ root: this.project.root });
```

Comment:
generally speaking, we should do `new VersionChecker(this.project)` but this should work just fine

Reply:
I swear I tried that and had test failures, but now I went back to prove it and it works fine. 🤷‍♂️